packages/perseus/src/perseus-types.ts

```diff
@@ -664,6 +664,17 @@ export type LockedPointType = {
     filled: boolean;
 };
 
+export type LockedLineType = {
+    type: "line";
+    kind: "line" | "ray" | "segment";
+    points: [startPoint: LockedPointType, endPoint: LockedPointType];
+    color: string;
+    lineStyle: "solid" | "dashed";
```

Comment:
**nit**: since this is in a locked line, maybe this can just be `style`?  Although, using `lineStyle` will allow us to be consistent when we do circle and polygon.  Ignore me. :)

Reply:
I think the reason I did `lineStyle` here is that it's not a style type, it's just two possible values. I didn't want it to be confused with svg style props.